fv3core/stencils/remapping_part1.py

```diff
@@ -299,11 +299,11 @@ def __call__(
         hs: FloatFieldIJ,
         te: FloatField,
         ps: FloatFieldIJ,
-        wsd: FloatField,
+        wsd: FloatFieldIJ,
         omga: FloatField,
         ak: FloatFieldK,
         bk: FloatFieldK,
-        gz: FloatField,
+        gz: FloatFieldIJ,
```

Comment:
I don't know if there is upcasting/downcasting of fields/storages, but I figure I mention just in case.  In `remapping.py`, line 51, `gz` is defined as a `FloatField`, and the call to `_moist_cv_pt_pressure` also has `gz` as a `FloatField`, but it's typed here as `FloatFieldIJ`.  Perhaps it'll still work, but I wanted to double check.

Reply:
good catch, this was an accident -- at first though gz was supposed to be 2d, but it's not